coalib/parsing/filters/Filter.py

```diff
@@ -0,0 +1,15 @@
+def filter(filter_applier):
+    def filter_wrapper(bears, args):
+        args = {arg.lower() for arg in args}
+        local_bears, global_bears = bears
+        local_bears = filter_section_bears(local_bears, args)
+        global_bears = filter_section_bears(global_bears, args)
+        return local_bears, global_bears
+
+    def filter_section_bears(bears, args):
```

Comment:
maybe put `filter_section_bears` before `filter_wrapper`? As you use it above "before it's defined" regarding reading order. Up to you :+1: